resolver/addressable_resolver.go

```diff
@@ -103,6 +113,33 @@ func (r *URIResolver) URIFromDestination(dest duckv1beta1.Destination, parent in
 	return "", fmt.Errorf("destination missing Ref, [apiVersion, kind, name] and URI, expected at least one")
 }
 
+// URIFromDestinationV1 resolves a v1.Destination into a URI string.
+func (r *URIResolver) URIFromDestinationV1(dest duckv1.Destination, parent interface{}) (string, error) {
+	if dest.Ref != nil {
+		url, err := r.URIFromObjectReference(dest.Ref, parent)
+		if err != nil {
+			return "", err
+		}
+		if dest.URI != nil {
+			if dest.URI.URL().IsAbs() {
```

Comment:
I thought we loosened this to just use `ResolveReference` when not nil and if the uri is a full rewrite, so be it.

Reply:
The old one still uses it above, so I didn't think so? I'd rather decouple that change from this though if that's cool?